### ISSUE TITLE
Android Notification Dose Not Close

### ISSUE THREAD

**Issue by rooneyl** [voxeet/voxeet-cordova-conferencekit#83]
**Describe the bug**

Android Notification Popup dose not disappear after accepting a call

**Expected Behaviour**

Notification Popup should be closed upon accepting a call.

**Steps to Reproduce the Problem**

  1. Create Plain Cordova Project (Cordova@11, Cordova-Android@10.1.2, Voxeet@1.5.9 or Voxeet@fix/issue_with_notifications)
  2. Enable Push Notification from Config.xml
  3. Add POST_NOTIFICATIONS for fix/issue_with_notifications branch.
  4. Initiate call from outside, confirm notification is coming and try accepting.

**Minimalistic code (recommended)**
- config.xml
```
        <preference name="android-minSdkVersion" value="26" />
        <preference name="android-targetSdkVersion" value="33" />
        <preference name="VOXEET_CORDOVA_USE_PUSH" value="true" />
        <preference name="GradlePluginGoogleServicesEnabled" value="true" />
        <resource-file src="google-services.json" target="app/google-services.json"/>
        <config-file parent="/manifest" target="app/src/main/AndroidManifest.xml" xmlns:android="http://schemas.android.com/apk/res/android">
            <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
        </config-file>
```

-www/js/index.js
```
await VoxeetSDK.initialize("*************", "*************")

const isUserLoggedIn = await VoxeetSDK.isUserLoggedIn()
if (isUserLoggedIn) {
    await VoxeetSDK.disconnect()
}

const id = "*************"
const avatar = "*************"
await VoxeetSDK.connect(new UserInfo(id, "Test", avatar))

```
**Specifications**

  - Device: Pixel 6
  - OS: Android 12
  - Platform: Android
  - Version:
    - cordova@11
    - cordova-android@10.1.2
    - voxeet-uxkit-cordova@1.5.9 or voxeet-uxkit-cordova#fix/issue_with_notifications

**Comment by codlab** [comment]
The branch fix/issue_with_notifications should be using the fix that as been released few minutes ago in the uxkit for android. Can you do the following just to check you will be using the new version:

```
cd platforms/android
./gradlew :app:dependencies | grep io.dolby | grep uxkit
```

You should see mentions of 3.8.4 👍

**Comment by codlab** [comment]
There's a transparent release for the underlying Android's UXKit which targets a crash happening when starting the app. Nothing to do or commit to change in your app, cleaning and rebuilding the app should be enough to use the new 3.8.5 👍

**Comment by rooneyl** [comment]
Thank you, it looks like all issues haven been resolved. :smiley:

**Comment by weiz18** [comment]
Hey, @codlab ,  we are getting this error when using the version on `voxeet-uxkit-cordova@1.5.9 or voxeet-uxkit-cordova#fix/issue_with_notifications` , any idea ? thanks ! 

  `Caused by: java.lang.RuntimeException: Duplicate class com.dolby.voice.devicemanagement.DeviceManagerSupport found in modules jetified-media-3.7.2-runtime (com.voxeet.sdk:media:3.7.2) and jetified-media-3.9.0-beta.1-runtime (io.dolby:media:3.9.0-beta.1)`

**Comment by codlab** [comment]
Do you use the push plugin ? Is it updated as well to the latest version available ? also can you do the following :

```
cd platforms/android
./gradlew :app:dependencies
```

**Comment by weiz18** [comment]
Yep, we are using the push plugin and the version is 0.1.0(https://www.npmjs.com/package/capacitor-plugin-voxeet-push). 
Isn't the error indicating there were two module importing the same class ? maybe the version mismatched ? i can see that one is using 3.7.2-runtime and the other one is 3.9.0-beta.1

**Comment by codlab** [comment]
Indeed the capacitor update was missing on npmjs ;) should be ok now (0.2.0)

**Comment by weiz18** [comment]
yep. all good now after upgrading. thanks !

**Comment by weiz18** [comment]
@codlab The build issue is all good now but i can't seem to receive any notification when the app is killed. i have the following line when i do a call, not sure if its gonna help 
`No bubble up: not allowed to bubble: 0`

but it looks like there is no log coming through and as soon as i opened the app, it opens the incoming call menu

**Comment by weiz18** [comment]
> Hey, @codlab , we are getting this error when using the version on `voxeet-uxkit-cordova@1.5.9 or voxeet-uxkit-cordova#fix/issue_with_notifications` , any idea ? thanks !
> 
> `Caused by: java.lang.RuntimeException: Duplicate class com.dolby.voice.devicemanagement.DeviceManagerSupport found in modules jetified-media-3.7.2-runtime (com.voxeet.sdk:media:3.7.2) and jetified-media-3.9.0-beta.1-runtime (io.dolby:media:3.9.0-beta.1)`

i think this issue is back on the latest version of plugin. i had to force the version of uxkit inside our gradle file to fix the error  `voxeetUXKitVersion = '3.9.0'`. 🙏   maybe we need to address inside capacitor push plugin again ? thanks !
